src/00/01-rfc-process/proposal.adoc

```diff
@@ -0,0 +1,280 @@
+times. But we allow for flexibility in the process. The goal is to reduce
+friction as much as possible without sacrificing comprehensive evaluation of
+RFCs.
+
+[#guidelines-mechanics]
+=== Mechanics
+
+Do not squash, or otherwise coalesce, changes for RFC PRs::
+After creating an RFC PR additional updates should be done as additional, plain,
+commits. Having the additional commits helps in tracking how feedback is
+addressed. When an RFC is approved it will be squash merged. Hence attempts to
+pre-squash would be wasted.
+
+Mirror external feedback in the PR::
+Any feedback you get outside of PR comments should be summarized, by you, as
+comments in the PR. And if there are changes, i.e. additional commits, to the
+RFC that are related they should be mentioned in the PR comments.
+
+Do not change approved PRs::
+After a PR is approved, and labeled as such, do not alter it. It will be merged
+in due time. If there are followup changes they should come as different RFCs.
+
+[#guidelines-design]
+=== Design
+
+Prefer solutions that cast a wide net::
+The key goal of EcoStd is to facilitate wide adoption and interoperation. Hence
+single solutions that solve a wide scope are preferred. Which means that you can
+expect questions like: "`Does this solve X, Y, and Z also?`", "`Have you
+considered use case U?`", and more.
+
+Including existing practice is essential for an RFC::
+Engineering is an iterative practice as such it makes progress from existing
+solutions towards better ones. Solutions without a problem are just dreams that
+are not likely widely shared. For those reasons, and more, it's imperative that
+an RFC is based on existing practice, i.e. demonstrate that there's a problem
+and solution that this is trying to improve.
+
+Seek consensus with proposals and when responding to feedback::
+Because of the goal of this collective work is to achieve as widespread adoption
+as possible it's important to consider the varied viewpoints when composing
+solutions to problems. You should seek consensus as much as possible at all
+times. There are many ways to achieve consensus, and it would be impossible to
+enumerate them. But some general ones are:
+* Provide rationale (pros and cons) for decisions and feedback.
+* Seek to increase factual data in your proposal.
+* Include real world examples and use cases.
+* Provide before and after comparisons if you are proposing behavior changes.
+
+:leveloffset: -1
+
+[#prior-art]
+== Prior Art
+
+This proposal is roughly based on these processes:
+
+https://github.com/zeromq/rfc[ZeroMQ RFC]::
+The ZeroMQ process demonstrates how an RFC process can be applied to
+specifications.
+
+https://github.com/rust-lang/rfcs[Rust Language RFC]::
+The Rust RFC process provides a compelling demonstration of the ability to
+agree on changes to the Rust language in an open manner.
+
+Additional influence comes from:
+* https://www.boost.org/community/reviews.html[Boost C++ Libraries review process].
```

Comment:
I'm don't know adoc offhand to suggest a fix, but this isn't rendering as a link, which I believe is the intention. Also it's not being rendered as bullet points.